app/waf/src/controllers/ping.controller.ts

```diff
@@ -1,5 +1,9 @@
 import {Request, RestBindings, get, ResponseObject} from '@loopback/rest';
 import {inject} from '@loopback/context';
+import {AS3Service} from '../services';
+
+const AS3_HOST: string = process.env.AS3_HOST || 'localhost';
+const AS3_PORT: number = Number(process.env.AS3_PORT) || 8443;
```

Comment:
We may have more configuration items in future, just thinking if we could separate them to a json-file datasource/repository/model/(controller if necessary for configurable from api) for all configuration?